tests/tracking/fluent/test_fluent.py

```diff
@@ -83,10 +83,12 @@ def create_run(
 
 
 def create_test_runs_and_expected_data(experiment_id=None):
-    """Create a pair of runs and a corresponding data to expect when runs are searched
-    for the same experiment
+    """
+    Create a pair of runs and a corresponding data to expect when runs are searched
+    for the same experiment.
 
-    :return: (list, dict)
+    Returns:
+        A tuple containing a list and a dictionary.
```

Comment:
Reasonable improvement

Reply:
Can we say `a tuple of a list and a dictionary`? `contain` sounds like it contains something else.